### ISSUE TITLE
 16 binary system

### ISSUE THREAD

**Issue by air798** [RVillani/UE4Duino#9]
Hello, will the plug-in support the sending of hexadecimal strings in the future? If you can, it will be the only useful serial port plug-in for UE4!

**Comment by RVillani** [comment]
Do you mean sending them as String (which is already possible, but Arduino will also get them as String) or a node that converts the hexa string to a byte? That sounds interesting.

**Comment by StephenSohng** [comment]
If I may ask, when you say "sending them as String (which is already possible", what do you mean by that? Can you send Strings directly? (I currently only see the ability to write floats, integers, and bytes) (please tell me if I'm missing something)

**Comment by RVillani** [comment]
Yeah. There are the Print and Print Line functions, which you use to send Strings to Arduino (without and with a line end, consecutively).
If you want to send hexa as numbers, that's just bytes. You just need to know how to write such hexas as decimal values.
But if you want to send the actual characters, like "2F", use Print or Print Line.
If you drag a line from a Serial variable and search for write (or string), the Print functions will show up, because they are tagged to do so.

**Comment by StephenSohng** [comment]
Ahh, that explains so much. Thank you. This helps tremendously.
